### DISCUSSION TITLE
Problem with skin texture

### DISCUSSION THREAD

**Discussion by iamhardikjain** [lllyasviel/Fooocus#2192] [Q&A]
Hi , huge fan for your work sir. Kindly help me with solving problem for changing skin texture as it won't change even after changing/trying multiple attempts

**Comment by lgwjames** [comment]
example

**Reply by iamhardikjain** [reply]
am trying to generate  ai model but i want to control skin colour of the model. I was trying to make a Indian influencer but as soon word indian strikes model automatically goes dark skin texture ( if india word ignored skin is white )

**Reply by xhoxye** [reply]
You want to generate a white Indian, right?
